.drone.yml

```diff
@@ -3,7 +3,10 @@ kind: pipeline
 type: kubernetes
 name: DCU
 trigger:
+  branch:
+    - master
   event:
+    - pull_request
```

Comment:
Please don't do anything expensive or scary on a `pull_request` -- anyone with a GitHub account can open a pull request against this repo.